pkgs/development/libraries/gstreamer/core/default.nix

```diff
@@ -41,18 +42,20 @@ stdenv.mkDerivation rec {
     docbook_xsl docbook_xml_dtd_412
   ];
 
-  buildInputs =
-       lib.optionals stdenv.isLinux [ libcap libunwind elfutils ]
-    ++ lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.CoreServices;
+  buildInputs = [ libunwind ]
+    ++ lib.optionals stdenv.isLinux [ libcap elfutils ]
+    ++ lib.optional stdenv.isDarwin CoreServices;
 
   propagatedBuildInputs = [ glib ];
 
   mesonFlags = [
     # Enables all features, so that we know when new dependencies are necessary.
-    "-Dauto_features=enabled"
+    # "-Dauto_features=enabled"
```

Comment:
Could you disable just the feature you do not want on darwin?

Reply:
I'm not sure how, but my changes are all darwin only.

Reply:
This one is not.

Reply:
Oh, whoops. I didn't mean to include that.

Reply:
Do you know if there's a way to disable the pkgconfig checks for a dependency? The other part didn't do anything.

Reply:
Other than patching it, I do not think so.

You can disable it by using

`-Dunwind=disabled`

as per https://gitlab.freedesktop.org/gstreamer/gstreamer/blob/c34fe82b5badf3c82d81f5baa4187d599aa6f9df/meson_options.txt#L22

but that will also remove unwind support on Darwin. If you want to support that, you can replace the following line by

https://gitlab.freedesktop.org/gstreamer/gstreamer/blob/c34fe82b5badf3c82d81f5baa4187d599aa6f9df/meson.build#L349


```meson
unwind_dep = cc.find_library('unwind', required : get_option('libunwind'))
```

on Darwin.